tests/test_distortion.py

```diff
@@ -61,8 +65,12 @@ def testDistortion(self):
             self.fail("Cannot find saved data %r; set SAVE_DATA = True and run again to save new data" %
                       dataPath)
 
+        fieldAngleToFocalPlaneTolerance = transformRegistry["hsc"].ConfigClass().tolerance
```

Comment:
I think it would be better to put a hard number here rather than using a `Config` value which could change.

Reply:
The value I'm testing against is the value used to determine the quantity being tested -- the default value of `tolerance` in the config. Yes if somebody changes that default then the unit test might fail, but that change would have to be done in this package, so the person who changed the default will see the problem right away and be able to deal with it.